plugins/source/bitbucket/testdata/config.yml

```diff
@@ -1,6 +1,6 @@
 kind: source
 spec:
-  name: bitbucket
+  name: tables
```

Comment:
this doesn't look right :)

Reply:
That's actually on purpose. The docs command uses the name attribute for the name of the directory the docs will be generated to